try {
    // Code that might throw an error
    throw new Error('Something went wrong!');
  } catch (error) {
    // Handling the error
    console.log(error.message); // Output: Something went wrong!
  }
  